### PR TITLE
Mainstems

### PR DESCRIPTION
**Pull request by dblodgett-usgs**
Adding mainstems to the ref namespace.

### CODE REVIEW

File: pygeoapi/pygeoapi.config.yml [hunk 1]
```diff
@@ -319,6 +319,43 @@ resources:
...
+        title: Reference Mainstems
+        geojsonld: false
+        description: US Reference Mainstem Rivers
+        keywords:
+            - Mainstem
+        context:
+            - schema: https://schema.org/
+              type: "@type"
+              name_at_outlet: schema:name
+              head_nhdpv2_COMID: https://www.wikidata.org/wiki/Q7376362
```
Comment (dblodgett-usgs) [inline pygeoapi/pygeoapi.config.yml:333]: Is this at all the right way to do this?

Comment (ksonda) [reply pygeoapi/pygeoapi.config.yml:333]: you're going to want to do something like:

```
- schema: https://schema.org/
  name_at_outlet: schema:name
- wiki: https://www.wikidata.org/wiki/
  head_nhdpv2_COMID: 
     "@id": wiki:Q7376362
     "@type": "@id"

```

**Comment by ksonda** [comment]
Also please add curl-able URL for gpkg to Dockerfile

File: pygeoapi/pygeoapi.config.yml (hunk 1 above)
Comment (ksonda) [reply pygeoapi/pygeoapi.config.yml:333]: any predicate whose object you want to be a URI you have to specify in that dumb nested way where it's "@type" (object type) is "@id"

Comment (dblodgett-usgs) [reply pygeoapi/pygeoapi.config.yml:333]: Poor `@type` and `@id` users... lol 

I'll fix this up now.

**Comment by ksonda** [comment]
last issue. gpkg has ids like /ref/mainstem/<numbers>, pid request has /ref/mainstems/<numbers>

**Comment by dblodgett-usgs** [comment]
Fixed an uploaded to hydroshare. https://github.com/dblodgett-usgs/geoconnex_prep/commit/e64ca8255e032441a213b9abd27315ee3c8014a2

**Pull request merged**